features/admin.actions.v1/pages/actions.scss

```
@@ -81,33 +81,86 @@
+       
+
+
+                .icon-status {
+                    color: #D35400;
+                    font-size: 0.8em;
+                    width: 1em;
+                    height: 1em;
+                    margin-right: 3.5px;
+                    stroke-width: 1.3px;
+                    stroke: #D35400;
+                    fill: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)